### ISSUE TITLE
Page State from docs lights up all apprailtitles

### ISSUE THREAD

**Issue by piperun** [skeletonlabs/skeleton#1451]
### Current Behavior

Page State from docs lights up all apprailtitles instead of just one as it's suppose to according to this in the documentation:
https://www.skeleton.dev/components/app-rail#page-state
```
<AppRailTile tag="a" href={tileUrl} class="{tileUrl === $page.url.pathname ? '!bg-primary-500' : ''}">
    (icon)
</AppRailTile>
```
However with a more real-life example:
```js
    <AppRail selected={storeValue}>
        <AppRailTile label="Home" title="Home" tag="a" href="/">
(icon)
        </AppRailTile>

        <AppRailTile
            label="Create"
            tag="a"
            href={'/path/create'}
            title="Create"
            class={'/path/create' === $page.url.pathname ? '!bg-primary-500' : ''}
        >
(icon)
        </AppRailTile>
        <AppRailTile
            label="View"
            tag="a"
            href={'/path'}
            title="View"
            class={'/path' === $page.url.pathname ? '!bg-primary-500' : ''}
        >
(icon)
        </AppRailTile>
    </AppRail>
```
    It'll light up everything as soon as you click on an `<AppRailTile>`.

### Reproduction / Steps To Reproduce

1. Follow the guide: https://www.skeleton.dev/components/app-rail#page-state
2. use example: 
```js
    <AppRail selected={storeValue}>
<AppRailTile tag="a" href={"/path/one"} class="{/path/one === $page.url.pathname ? '!bg-primary-500' : ''}">
    (icon)
</AppRailTile>
<AppRailTile tag="a" href={"/path/two"} class="{/path/two === $page.url.pathname ? '!bg-primary-500' : ''}">
    (icon)
</AppRailTile>
    </AppRail>
```

3.  click on an `<AppRailTile>` and it should light up all the `<AppRailTile>`

**Comment by Mahmoud-zino** [comment]
This is fixed by #1402 and is included in the next release.

**Comment by endigo9740** [comment]
See also:
- https://github.com/skeletonlabs/skeleton/issues/1478
- https://github.com/skeletonlabs/skeleton/pull/1486
